src/bin/psql/adhoc_compat.h

```diff
@@ -0,0 +1,31 @@
+#ifndef ADHOC_COMPAT_H_028E0843
+#define ADHOC_COMPAT_H_028E0843
+
+#include <unistd.h>
+
+void die(const char* s);
+
+void
+ExceptionalCondition(const char *conditionName,
+					 const char *errorType,
+					 const char *fileName,
+					 int lineNumber);
+
+void elog_start(const char *filename, int lineno, const char *funcname);
+void elog_finish(int elevel, const char *fmt,...);
+
```

Comment:
This was a leftover from pulling in the backend rbtree. Not needed anymore, I will kill them off.